packages/ember-simple-auth/tests/unit/mixins/application-route-mixin-test.js

```diff
@@ -29,6 +31,7 @@ describe('ApplicationRouteMixin', () => {
       clear: sinon.stub()
     }));
 
+    this.owner.register('route:application', Route.extend(ApplicationRouteMixin));
```

Comment:
why is this change necessary now?

Reply:
This was previously relying on the application route that's defined in the dummy app. The dummy app of `packages/ember-simple-auth` is now empty though as it has been moved to `packages/test-app`, thus there is no application controller anymore if we don't register one.